src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -844,7 +844,7 @@ def load_arguments(self, _):
         c.argument('version', help='image version')
         c.argument('end_of_life_date', help="the end of life date, e.g. '2020-12-31'")
         c.argument('storage_account_type', help="The default storage account type to be used per region. To set regional storage account types, use --target-regions",
-                   arg_type=get_enum_type(["Standard_LRS", "Standard_ZRS"]), min_api='2019-03-01')
+                   arg_type=get_enum_type(["Standard_LRS", "Standard_ZRS", "Premium_LRS"]), min_api='2019-03-01')
```

Comment:
Is this defined in `SnapshotStorageAccountTypes` enum? 
Maybe we can use `get_enum_type(SnapshotStorageAccountTypes)` to avoid such problems.

Reply:
You are right.

Reply:
SDK does not have this type. Swagger does not, either. However, it can be accepted by service.